libs/react-spa/bff/src/lib/bff.hooks.ts

```diff
@@ -46,6 +46,8 @@ export const useUserBirthday = () => {
   return useMemo(() => {
     const nationalId = userInfo?.profile.nationalId
 
+    // This does not support system nationalId(kerfis kennitala) users.
+    // This should be read from the parsed id_token in the /user response, when the IDP adds support for it.
```

Comment:
~Should be part of the `useUserBirthday` documentation so people see it in their IDE when calling the function.